src/components/Badge/BadgeReport.jsx

```diff
@@ -245,6 +247,12 @@ const BadgeReport = props => {
     //Reload the view profile page with updated bages
     window.location.reload();
   };
+  console.log(props.badges[1].earnedDate.length);
+  let arr = [];
```

Comment:
Please remove your new line 250-255. Looks like the `arr` only used in this snippet of code. Please correct me if I'm wrong. 
<img width="653" alt="image" src="https://user-images.githubusercontent.com/5071040/212199358-c5a192ae-cdb5-4dba-8947-82b6cddb8a85.png">

Reply:
Yes you are right, removed!